lib/registration.pm

```diff
@@ -260,8 +260,8 @@ sub fill_in_registration_data {
             assert_screen('scc-beta-filter-checkbox');
             send_key('alt-i');
         }
-        elsif (!check_screen($modules_needle)) {
-            record_soft_failure('Modules are still beta until GM or FCS');
+        elsif (!check_screen($modules_needle, 0)) {
```

Comment:
I think I also stated that we should use multi tag assert screens. This will most likely not work as the previous send_key might not have resulted in an action yet

Reply:
@okurz please, see our discussion in PR #5124

Reply:
@okurz, please see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5142